packages/run-protocol/src/econ-behaviors.js

```diff
@@ -281,18 +281,19 @@ export const startVaultFactory = async (
     ]);
   const ammPublicFacet = await E(zoe).getPublicFacet(ammInstance);
   const feeMintAccess = await feeMintAccessP;
-
+  const pa = await priceAuthority;
+  const timer = await chainTimerService;
```

Comment:
we could be awaiting all those in the `Promise.all` to start in parallel but that does obscure readability.

I figure this whole file will get some cleanup soon.